common/src/main/java/dev/ftb/mods/ftbchunks/client/gui/SliceCreationGUI.java

```diff
@@ -0,0 +1,362 @@
+package dev.ftb.mods.ftbchunks.client.gui;
```

Comment:
This file is pretty messy, please clean up the code and add some comments here and there to explain the more weird sections